specs/permissions.md

```diff
@@ -0,0 +1,340 @@
+
+### excludedProperties
+The "includedProperties" member is an array of strings that identify properties of the resource representation returned by the path, that are not accessible with the permission.
+
+
+## <a name="schemeObject"></a>Scheme Object
+The scheme object has members that describe the permission within the context of the scheme. Additional members provide behavioral constraints of the permission when used with the scheme.  
+
+```json
+"schemes": {
+    "DelegatedWork": {
+        "adminDisplayName": "Read and write app activity to users'activity feed",
+        "adminDescription": "Allows the app to read and report the signed-in user's activity in the app.",
+        "consentDisplayName": "Read and write app activity to users'activity feed",
+        "consentDescription": "Allows the app to read and report the signed-in user's activity in the app.",
+        "requiresAdminConsent": true
+    },
+    "DelegatedPersonal": {
+        "consentDisplayName": "Read and write app activity to users'activity feed",
+        "consentDescription": "Allows the app to read and report the signed-in user's activity in the app."
+    },
+    "Application": {
+        "adminDisplayName": "Read and write app activity to users' activity feed",
+        "adminDescription": "Allows the app to read and report the signed-in user's activity in the app.",
+    }
+```
+
+### adminDisplayName
+The "adminDisplayName" member is a string that provides a short permission name that considers the current scheme and the perspective of a resource administrator.
+
+### adminDescription
+The "adminDescription" member is a string that describes the permission considering the current scheme from the perspective of a resource administrator.
+
+### consentDisplayName
+The "consentDisplayName" member is a REQUIRED string that provides a short permission name that considers the current scheme and the perspective of the user consenting an application.
+
+### consentDescription
+The "consentDescription" member is a REQUIRED string that describes the permission considering the current scheme from the perspective of the user consenting an application.
+
+### requiresAdminConsent
+The "requiresAdminConsent" member is a boolean value with a default value of false. When true, this permission can only be consented by an adminstrator.
+
+## <a name="pathObject"></a>Path Object
+The path object contains properties that affect how the permission object controls access to resource identified by the key of the path object.
+
+```json
+"paths": {
+  "/me/activities/{id}": {
+    "leastPrivilegePath": ["DelegateedWork", "DelegatedPersonal"],
+    "includedProperties": ["id","displayName"],
+    "excludedProperties": ["cost"]
+  }
+```
+
+### leastPrivilegePath
+The "leastPrivilegePath" member is an array of strings that identify the schemes for which this permission is the least privilege permission for accessing the path.
```

Comment:
Question: can this not be derived automatically based on the set of paths and the **privilegeLevel** property?

Reply:
That is what I originally thought and then someone convinced me that is not the case.  Let me try and remember what the reasoning was.  Nope, can't remember.  I will go back to the person and get them to remind me and I'll update this doc.  Thanks for the call out.